docs/locale/ja/LC_MESSAGES/gradle.po

```diff
@@ -76,14 +88,4 @@ msgid ""
 msgstr "たとえば，SpotBugs のバージョンを指定したいときは，次のように設定します。"
 
 #~ msgid ""
-#~ "This Gradle Plugin introduces two tasks:"
-#~ " `spotbugsMain and `spotbugsTest."
 #~ msgstr ""
-
-#~ msgid ""
-#~ "SpotBugs Gradle Plugin adds task "
-#~ "dependency from `check` to these these"
-#~ " tasks, so you can simly run "
-#~ "``./gradlew check`` to run SpotBugs."
-#~ msgstr ""
-
```

Comment:
This file contains many meaningless diff, but they can be ignored.

@orihalcon128 I guess they comes from the difference between my Babel version and yours. Can I ask how you generate .po files? Not by `docs/build.sh`?

Reply:
I am translating po files using OmegaT.

1. make gettext
2. sphinx-intl update -p .build/locale -l ja
3. Copy po files to source folder of OmegaT project.
4. Open the po file with OmegaT and translate it.
5. Generate the translated files.
6. Copy po files in target folder of OmegaT project to docs/locale/ja/LC_MESSAGES folder.

For more information on this see http://omegat.sourceforge.net/manual-standard/en/chapter.instant.start.guide.html

Reply:
Got it, thanks, it should have no problem.